.github/workflows/Release.yml

```diff
@@ -0,0 +1,14 @@
+name: Build Package to use ncc
+on:
+  push:
+    branches:
+      - master
+jobs:
```

Comment:
Is this really needed? We will anyway have this file as part of the PR, right?

Reply:
> @Purushothaman22 we still need `dist` to be checked in right? This PR removes it.

@gps  I removed it intentionally to check if it is working. 

> Is this really needed? We will anyway have this file as part of the PR, right?

Not needed will remove it.